### PR TITLE
Add project skeleton for v2

### PR DESCRIPTION
**Pull request by schana**
This removes v1 of the solution and prepares the skeleton for development to commence on v2.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/unit-test-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - uses: actions/checkout@v3
+      - name: ${{ matrix.os }} ${{ matrix.python }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python }}
+          cache: 'pip'
+          cache-dependency-path: 'pyproject.toml'
+      - name: Install Tox and any other packages
+        run: |
+          pip install --upgrade pip
```
Comment (hearde) [inline .github/workflows/unit-test-workflow.yml:25]: <nit> why not upgrade setuptools here as well?

File: source/refreezer/app.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+from refreezer.infrastructure.stack import RefreezerStack
+from refreezer.pipeline.stack import PipelineStack
+
+
+def main() -> None:
+    app = cdk.App()
+    RefreezerStack(app, "refreezer")
+    PipelineStack(app, "pipeline")
+    app.synth()
```
Comment (hearde) [inline source/refreezer/app.py:16]: Curious, why synth here explicitly?

File: source/refreezer/pipeline/stack.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+from refreezer.infrastructure.stack import RefreezerStack
+
+
+class PipelineStack(Stack):
+    """
+    This stack establishes a pipeline that builds, deploys, and tests the solution
+    in a specified account. It also uses CodeStar connections to set up a webhook
+    to GitHub to trigger the pipeline when commits are pushed.
+
+    The repo is configured using SSM parameters, specifically the following:
```
Comment (hearde) [inline source/refreezer/pipeline/stack.py:27]: Is this a standard way to implement a cdk pipeline?
Does this mean that you won't rely on internal pipelines?
I'm excited to hear if you recommend this process for other solutions.

**Review by hearde — APPROVED** [review]

File: source/refreezer/pipeline/stack.py (hunk 3 above)
Comment (schana) [reply source/refreezer/pipeline/stack.py:27]: This is mostly standard in using the cdk pipelines construct, but is a little different in that it adds an integration test phase after the deploy.

**Review by evanwieren — APPROVED** [review]

**Pull request merged**